app/containers/AccountInitializer/reducer.js

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * AccountInitializer reducer
+ *
+ */
+
+import {
+  REVIEW_ACCOUNT,
+  REVIEW_ACCOUNT_SUCCESS,
+  REVIEW_ACCOUNT_ERROR,
+} from './constants';
+
+export const initialState = {
+  loading: false,
+  error: null,
+};
+
+function accountInitializerReducer(state = initialState, action) {
+  const { type, err, acc } = action;
+  switch (type) {
+    case REVIEW_ACCOUNT:
+      return {
```

Comment:
Return state and update state using setState function. See example for details https://github.com/react-boilerplate/react-boilerplate/blob/master/app/containers/HomePage/reducer.js